### ISSUE TITLE
Can't execute npm scripts : 'Error: Cannot find module "Immutable"'

### ISSUE THREAD

**Issue by chprome** [adazzle/react-data-grid#187]
Hello,

I can't run any npm scripts (`npm test`, `npm debug-test` nor `npm example`).
I get the following error :

```
/src/__tests__
 @ ./src/__tests__/ColumnMetrics.spec.js 5:35-55
27 01 2016 22:02:46.684:INFO [karma]: Karma v0.13.19 server started at http://localhost:9876/
27 01 2016 22:02:46.692:INFO [launcher]: Starting browser PhantomJS
27 01 2016 22:02:47.309:INFO [PhantomJS 1.9.8 (Linux 0.0.0)]: Connected on socket /#ubRH31JoaKOkq4LxAAAA with id 66485614
PhantomJS 1.9.8 (Linux 0.0.0) ERROR
  Error: Cannot find module "Immutable"
  at /home/christophe/dev/projects/react-data-grid/test/unitTests.jsx:22492
```

I have a fresh clone of the project, on master branch.

node version is 5.5.0, 
npm version is 3.3.1

I am on Linux (Ubuntu Gnome 15.10)

Anyone having the same issue ?
Any hint ?

Thanks
Christophe

**Comment by txm** [comment]
I don't have a test dir ...

node_modules/react-data-grid$ ls -1
LICENSE
README.md
addons.js
dist/
index.js
node_modules/
package.json
themes/

**Comment by txm** [comment]
Might be linked to? #80

**Comment by bakesteve** [comment]
yeah - while this shouldnt throw (and would be caught by our CI server which does a full fresh build each commit + every day - to catch any breaking changes in any of the node modules we use) it does sound related to #80 
have you tried `npm install` then `gulp`?
and maybe try `npm isntall Immutable` 1st?

**Comment by bakesteve** [comment]
ps we use appveyor for CI so its win based. If anyone has travis experience, be good to have that running too so we can ensure no x-platform issues

**Comment by bakesteve** [comment]
fixed in #281
